### ISSUE TITLE
Markup for examples with parts

### ISSUE THREAD

**Issue by sean-fitzpatrick** [APEXCalculus/APEXCalculusPTX#178]
In #177 I've updated a number of `<ol>` environments that were using `label="a"` because the syntax has changed to `marker="a"`.

But do we want to use `ol` at all for examples with multiple parts? Or should we restructure with `task`? (This is probably a question for @Alex-Jordan )

Right now we have a lot of examples structured like:
```
<example>
  <statement>
    <p>
      Do the following things:
    </p>
    <p>
      <ol marker="a">
        <li>The first thing</li>
        <li>The last thing</li>
      </ol>
    </p>
  </statement>
  <solution>
    <p>
      <ol marker="a">
        <li>
          <p>
            Solution to the first thing
          </p>
        </li>
        <li>
          <p>
            Solution to the last thing
          </p>
        </li>
      </ol>
    </p>
  </solution>
</example>
```

Would it make more sense for these to be written like this? Or should we keep things as they are?

```
<example>
  <introduction>
    <p>
      Do the following things:
    </p>
  </introduction>
  <task>
    <statement>
      <p>
        The first thing
      </p>
    </statement>
    <solution>
      <p>
        Solution to the first thing
      </p>
    </solution>
  </task>
  <task>
    <statement>
      <p>
        The last thing
      </p>
    </statement>
    <solution>
      <p>
        Solution to the last thing
      </p>
    </solution>
  </task>
</exercise>
```

**Comment by Alex-Jordan** [comment]
If it's high enough on the list of things to do, then I think such things should be structured with `task`, in general. I am unsure how good an idea it would be to automate converting them all. Occasionally, if the list within an example really is just a list (as opposed to a sequence of multiple sub-exercises), it should stay that way.

If I wanted to automate that anyway, I wouldn't feel comfortable doing it with text, regex, etc. Instead I'd try to use XSL. Start with an identity stylesheet that leaves everything alone. Then change how it acts towards `example` elements that have a `statement` that contains an `ol`. And you'd have to apply that stylesheet to each of the .ptx section files one at a time, not to the root file. And after doing all that, carefully review a diff for quality control.

**Comment by sean-fitzpatrick** [comment]
Yeah, thinking about it for awhile, I don't think I could do this without messing it up.
I'm going to look to see just how many of these there are.

**Comment by sean-fitzpatrick** [comment]
There are 93 examples that have parts in an `ol`. 69 of these have videos.

I'm probably not getting that done in a day!

**Comment by sean-fitzpatrick** [comment]
I am going to close this. PreTeXt requires that we use tasks in an exercise, but not in an example, so this is not high priority.
